Documentation/gettinghelp.rst

```diff
@@ -18,4 +18,142 @@ If you are interested in contributing to the code or docs, ping us on
 `Slack <https://cilium.herokuapp.com>`_ or just dive in on
 `Github <https://github.com/cilium/cilium/>`_!
 
+**************
+CLI Cheatsheet
+**************
+
+Below is a short list of examples of the various commands Cilium has to offer.
+If a command you use is missing please don’t hesitate to add it to one of the
+groups or create a new one.
+
+Basics
+======
+
+::
+
+        # Check the status of the agent
+
+        cilium status
+
+::
+
+        # Get the configuration knobs
+
+        cilium config
+
+Policy management
+=================
+
+::
+
+	# Importing a policy
+
+	cilium policy import <my-policy.json>
+
+::
+
+	# Get the latest imported policy
+
+	cilium policy get
+
+::
+
+	# Remove all the loaded policies
+
+	cilium policy delete --all
+
+Monitoring
+==========
+
+::
+
+	# Simple monitor output
+
+	cilium monitor 
+
+::
+
+	# Verbose output
+
+	cilium monitor -v
+
+::
+
+        # Filter for only the events to endpoint
+
+        cilium monitor --related-to=<id>
+
+::
+
+	# Filter only the dropped packets
+
+	cilium monitor --type drop
+
+::
+
+	# Dissect the payload
+
+	cilium monitor -v --hex
+
+Endpoints
+=========
+
+::
+
+	# Get a list of all the endpoints
+
+	cilium endpoint list
+
+::
+
+        # Get the endpoint model
+
+        cilium endpoint get <id>
+
+::
+
+        # Show log entries for endpoint
+
+        cilium endpoint log <id>
+
+::
+
+	# Enable debugging output for the endpoint
+
+	cilium endpoint config <my-ep> Debug=true
+
+Tracing
+=======
+
+::
+
+	# Trace between two applications on port 80 for connectivity issues
+
+	cilium policy trace -s <from> -d <to> --dport 80
```

Comment:
This should have three examples:
1. `--src-endpoint 200 --dst-endpoint 300 --dport 80`
2. `--src-k8s-pod namespace1:pod_foo --dst-k8s-pod namespace2:pod_bar`
3. `--src-identity 10 --dst-identity 20`

Reply:
Got to all of your comments except this one. Want to try out the commands before adding them. Will respin when I get the chance tomorrow. Thanks.